charts/galoy/values.yaml

```diff
@@ -289,9 +289,10 @@ galoy:
         timeoutSeconds: 1
   ## Kratos
   kratos:
-    masterPhonePasswordExistingSecret:
-      name: kratos-master-password
-      key: password
+    existingSecret:
+      name: kratos-secret
+      password: password
```

Comment:
```suggestion
      passwordless-password: passwordless-password
```

Reply:
not sure what is a good name for this, but I think password is similarly not descriptive enough. `passwordless-passord` or `master-client-password` are 2 options that would be a bit more descriptive.

Reply:
I don't quite understand this suggestion

Reply:
Fixed in 4772731e004d3bcf34a82395998432374e7bfa80